src/status_im/contexts/communities/overview/style.cljs

```diff
@@ -21,6 +21,9 @@
    :align-items    :center
    :margin-top     20})
 
+(def tmp-fetching
```

Comment:
Should we use something like `community-fetching-screen` or `community-fetching-placeholder`?